ONSdigital/dp-frontend-renderer
# Added meta tag for service name

### What

On error pages the service name is now displayed in a meta tag in the head of the page

### How to review

1. Checkout this branch and load the renderer local
2. Launch the web page and obtain an error page (easiest to accomplish when not running other services.
3. Open the page inspector (Developer console)
4. The following should be present <meta name="ons:service" content="dp-frontend-renderer">


### Who can review

A member of the team that is able run the dp-frontend-renderer service and open the dev tools in a browser.
Describe who worked on the changes, so that other people can review.
